cumulusci/core/keychain/base_project_keychain.py

```diff
@@ -28,12 +28,15 @@
 
 class BaseProjectKeychain(BaseConfig):
     encrypted = False
+    orgs: dict
+    app: object  # what is this?
+    services: dict
 
     def __init__(self, project_config, key):
         super(BaseProjectKeychain, self).__init__()
         self.config = {
             "orgs": {},
-            "app": None,
+            "app": None,  # what is this?
```

Comment:
It's where connected app info was kept before the concept of services was invented. I doubt it still needs to be here.